text-ui-test/EXPECTED.TXT

```diff
@@ -5,3 +5,89 @@ Hello from
 | |_| | |_| |   <  __/
 |____/ \__,_|_|\_\___|
 
+____________________________________________________________
```

Comment:
you can add these files to .gitignore as having there revision history doesnt add much value (consider this for input.txt as well)